### ISSUE TITLE
Downvoted and blocked

### ISSUE THREAD

**Issue by S4ltster** [Glass-Paramedic/Discord-tokens-bots-nukers-sel-bots#2]
I am offended and quite frankly, appalled at this.  its fine my beemovie script will beat your thing easily.

**FAQ**: What? Beemovie script? 

A: Yeah Beemovin on this cock. gottem good.

**Comment by Glass-Paramedic** [comment]
**1 FAQ: I just shit and cum at your comment**

**FAQ**
What does this mean?
The amount of shit (and cum) on my computer and floor has increased by one.

Why did you do this?
There are several reasons I may deem a comment to be worthy of feces or ejaculation. These include, but are not limited to:

Being gay


Am I going to shit and cum too?
No - not yet. But you should refrain from shitposting and cumposting like this in the future. Otherwise I will be forced to shit and cum again, which may put your shitting and cumming privileges in jeopardy.

I don't believe my comment deserved being shit and cum at. Can you un-cum it?
Sure, mistakes happen. But only in exceedingly rare circumstances will I put shit back into my butt. If you would like to issue an appeal, shoot me a hot load explaining what I got wrong. I tend to respond to retaliatory ejaculation within several minutes. Do note, however, that over 99.9% of semen dies before it can fertilize the egg, and yours is likely no exception.

How can I prevent this from happening in the future?
Accept the goopy brown and white substance and move on. But learn from this mistake: your behavior will not be tolerated in my mom's basement. I will continue to shit and cum until you improve your conduct. Remember: ejaculation is privilege, not a right.

**Comment by S4ltster** [comment]
> **1 FAQ: I just shit and cum at your comment**
> 
> **FAQ**
> What does this mean?
> The amount of shit (and cum) on my computer and floor has increased by one.
> 
> Why did you do this?
> There are several reasons I may deem a comment to be worthy of feces or ejaculation. These include, but are not limited to:
> 
> Being gay
> 
> Am I going to shit and cum too?
> No - not yet. But you should refrain from shitposting and cumposting like this in the future. Otherwise I will be forced to shit and cum again, which may put your shitting and cumming privileges in jeopardy.
> 
> I don't believe my comment deserved being shit and cum at. Can you un-cum it?
> Sure, mistakes happen. But only in exceedingly rare circumstances will I put shit back into my butt. If you would like to issue an appeal, shoot me a hot load explaining what I got wrong. I tend to respond to retaliatory ejaculation within several minutes. Do note, however, that over 99.9% of semen dies before it can fertilize the egg, and yours is likely no exception.
> 
> How can I prevent this from happening in the future?
> Accept the goopy brown and white substance and move on. But learn from this mistake: your behavior will not be tolerated in my mom's basement. I will continue to shit and cum until you improve your conduct. Remember: ejaculation is privilege, not a right.

**Caught you in 8K UHD surround sound 16 Gigs ram, HDR GEFORCE RTX, TI-80 texas insturments, Triple A duracell battery ultrapower100 Cargador Compatible iPhone 1A 5 W 1400 + Cable 100% 1 Metro Blanco Compatible iPhone 5 5C 5S 6 SE 6S 78 X XR XS XS MAX GoPro hero 1 2 terrabyte xbox series x Dell UltraSharp 49 Curved Monitor - U4919DW Sony HDC-3300R 2/3 CCD HD Super Motion Color Camera, 1080p Resolution Toshiba EM131A5C-SS Microwave Oven with Smart Sensor, Easy Clean Interior, ECO Mode and Sound On/Off, 1.2 Cu. ft, Stainless Steel HP LaserJet Pro M404n Monochrome Laser Printer with Built-in Ethernet (W1A52A) GE Voluson E10 Ultrasound Machine LG 23 Cu. Ft. Smart Wi-Fi Enabled InstaView Door-in-Door Counter-Depth Refrigerator with Craft Ice Maker GFW850SPNRS GE 28" Front Load Steam Washer 5.0 Cu.Ft. with SmartDispense, WiFi, OdorBlock and Sanitize and Allergen - Royal Sapphire Kohler K-3589 Cimarron Comfort Height Two-Piece Elongated 1.6 GPF Toilet with AquaPiston Flush Technology**

**Comment by S4ltster** [comment]
**For the record, I only need two buttons, Ctrl + C and Ctrl + V.**

**Comment by S4ltster** [comment]
**ACCORDING TO YOUTUBE STATISTICS**

**Trigger Warning**

Only a small percentage of my viewers are subscribed, so I beg in every video for you to subscribe, even though I gained already over 15 milllion subs in one year and every single Video of mine gets ten million of views at least. I make videos about 100% real Minecraft speed-runs and videos about man hunt where my friends hunt me and also can see my health at all times, so they conviently stop attacking me when I have only half a heart. I'm already hyper successful, so please, please subscribe.

**Comment by Glass-Paramedic** [comment]
`For the record, I only need two buttons, Ctrl + C and Ctrl + V.

`

Let's say, hypothetically, your mom was wearing a yellow raincoat. Now, also in this scenario, there is a man needing to get home. When he sees your mom, he will yell "taxi, taxi!". Now, why does he do this? The answer is actually quite simple. Your mom is so fat she is the size of a taxi, and the yellow raincoat she is theoretically wearing is the same color as a taxi. Therefore, the man mistakes her for a taxi. So, what has this hypothetical scenario shown us? It has shown us that your mom is very, very, fat. Boom! Once again destroyed with facts and logic.

**Comment by S4ltster** [comment]
> `For the record, I only need two buttons, Ctrl + C and Ctrl + V.
> 
> `
> 
> Let's say, hypothetically, your mom was wearing a yellow raincoat. Now, also in this scenario, there is a man needing to get home. When he sees your mom, he will yell "taxi, taxi!". Now, why does he do this? The answer is actually quite simple. Your mom is so fat she is the size of a taxi, and the yellow raincoat she is theoretically wearing is the same color as a taxi. Therefore, the man mistakes her for a taxi. So, what has this hypothetical scenario shown us? It has shown us that your mom is very, very, fat. Boom! Once again destroyed with facts and logic.

**Before I begin my actual comment, I would like to apologize in advance for my inadequate level of English proficiency.**


I am not a native speaker of the world's current lingua franca which unfortunately leads to me making numerous embarrassing mistakes being made whenever I attempt to communicate using this language. Whenever I am reminded of how I lack the ability to convey my thoughts in an eloquent manner, I feel as though I have committed a cardinal sin, as though every English teacher in the world is simultaneously shaking their head and sighing due to how utterly disappointed they are at me.

Although I know that saying sorry to those of you who are reading my comment will not change the fact that I fail miserably to write and speak perfect English, I am writing this as a way to deter a certain type of people who cannot stand poor English (Also known informally as Grammar Nazis) from mocking me by posting unwanted and unnecessary comments detailing my every blunder. In my humble opinion, making grammatical errors should be perfectly acceptable as native speakers should not expect non-native speakers to be able to communicate in their second or third languages eloquently. If you are able to completely understand what the other person wrote, is there really a problem with what they've written? No, because the entire concept of communication is the exchange of information between other intelligent beings, which means that no matter how the exchange of information is made, as long as the information is accurately shared there is not a fundamental issue with their ability to communicate. To see it in another way, remember that someone who isn't fluent in English is fluent in another language. When you think about it this way, isn't it impressive for someone to speak a second language in any capacity? Having empathy and respect are qualities that are sorely missing for far too many people these days, especially on the internet.

That being said, I am aware that not all netizens who correct others are doing it to ridicule and shame. There are some who do so with the intent to help others improve and grow. However, displaying the failures of other people publicly will cause the person who is criticized to feel negative emotions such as shame and sadness due to the fact that their mistake has been made obvious which severely undermines the point they were trying to make in spite of their unfamiliarity with the English language. In most circumstances people are not looking for language help when they post anything online. Most people just want to enjoy themselves and have a good time on the internet which is why I would not encourage correcting other people regardless of your intentions. If you really do want to help others with their spelling or grammar, I would highly recommend you to help via messaging privately because not only will you not embarrass anyone, you can also go more in-depth with your explanation which I'm sure the other person will greatly appreciate if they want help, but I digress. I know that I've written a bit of an essay, but I hope I've made my points clear.

**Comment by Glass-Paramedic** [comment]
>That's like 18 million words - Deltara

**Comment by S4ltster** [comment]
**AITA for kicking my creations out of my garden for eating an apple?**

_I know this already looks bad but hear me out._

I’m the immortal creator of the universe. Before all this drama started, I mostly hung out with these angels I created but I got bored of them and how they constantly worshipped me. Also they looked nothing like me, so I created this new beings in my image and I named him A. I really cared for A, I even threw out this other angel, L, who was disrespecting him.

When A told me he was feeling lonely, I created a wife for him. Let’s call her E. I let them live in my garden without paying rent. I told them specifically that they can eat all the garden fruit they wanted except this one apple.

So anyways, one day they did eat the apple. It was my apple, I had told them not to! So I threw them out on their naked asses. AITA?

**Edit: removed any identifying info**
